src/libsyntax/parse/literal.rs

```diff
@@ -4,7 +4,8 @@ use crate::ast::{self, Ident, Lit, LitKind};
 use crate::parse::parser::Parser;
 use crate::parse::PResult;
 use crate::parse::token::{self, Token};
-use crate::parse::unescape::{unescape_str, unescape_char, unescape_byte_str, unescape_byte};
+use crate::parse::unescape::{unescape_str, unescape_char, unescape_byte_str,
+    unescape_raw_str, unescape_byte};
```

Comment:
Nit: could you keep the "one line - one `use`" formatting.

Reply:
Done